.github/workflows/fastapi.yml

```diff
@@ -0,0 +1,35 @@
+name: fastapi-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  fastapi-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: fastapi
+    steps:
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: tiangolo/fastapi
+          ref: 0.63.0
+          path: fastapi
+      - name: Install Flit
+        run: pip install flit
+      - name: Install Dependencies
+        run: flit install --symlink
+      - name: Downgrade SQLalchemy
+        run: pip install --force-reinstall SQLAlchemy==1.3.24
```

Comment:
It would be nice if you could add a quick comment above each of these these steps to label as being fastapi specific, and maybe explain why these steps are necessary to provide some context in the future 😄